ci-operator/config/openshift/egress-router-cni/openshift-egress-router-cni-master.yaml

```diff
@@ -1,21 +1,31 @@
-binary_build_commands: make
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.15
+promotion:
+  name: "4.7"
+  namespace: ocp
 resources:
   '*':
     limits:
       memory: 4Gi
     requests:
       cpu: 100m
       memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
 tests:
-- as: unit
-  commands: make test-unit
-  container:
-    from: src
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
```

Comment:
`openshift_installer` is the old, deprecated template-based tests. The tests should now use the newer template-based tests ([doc](https://docs.ci.openshift.org/docs/architecture/step-registry/#ci-operator-test-configuration)) with the appropriate workflow (looks liks https://steps.ci.openshift.org/workflow/openshift-e2e-aws is the appropriate one - it runs the parallel [testsuite by default](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml#L10-L12))